### PR TITLE
Temporary set up for fixtures and firebase emulator

### PR DESCRIPTION
**Pull request by sevresbabylone**
This is a temporary set up for developing locally for ImageSwipe and WordSwipe which have different config.js files. They will be removed once the configuration/ styling / functionality for both games are finalised
ImageSwipe will be used for data annotation and WordSwipe will be used for assessment (with right and wrong answers)

To run this on local

1. [Install firebase emulator suite](https://firebase.google.com/docs/emulator-suite/install_and_configure). Make sure you have these installed
- [Node.js](https://nodejs.org/en/) version 8.0 or higher. (This project uses Node version 14 or higher)
- [Java](https://openjdk.java.net/install/) version 1.8 or higher.

2. Install the [Firebase CLI](https://firebase.google.com/docs/cli) via ```npm install -g firebase-tools```

3. Login into your firebase account via ```firebase login```. You can set any of your current firebase projects as active. 
4. Create a .env file at project root. uncomment depending on if you want to ruin ImageSwipe or WordSwipe
```
# VUE_APP_PROJECT_ID=wordswipe-test
# VUE_APP_PROJECT_ID=imageswipe-test
```
4. Start the firebase emulators using ```npm run start-emulator:imageswipe``` for ImageSwipe or ```npm run start-emulator:wordswipe```  before running ```npm run serve```. 
note: If you want to work on your own project with config.js which connects to the local emulator run ```firebase emulators:start``` before ```npm run serve``` but it won't have any data unless you import it.
5.  App should be running on localhost:8080

The below links will be redeployed with every commit.
[WordSwipe](https://swipes-for-science-demo.netlify.app/)
[ImageSwipe](https://imageswipe--swipes-for-science-demo.netlify.app/#/)

firebase-emulator/fixtures/sample-imageswipe-test/dbSeed.js 
firebase-emulator/fixtures/sample-wordswipe-test/dbSeed.js are used to create the fixtures but will be removed later.

### CODE REVIEW

File: firebase-emulator/fixtures/sample-imageswipe-test/saved-data/auth_export/accounts.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"kind":"identitytoolkit#DownloadAccountResponse","users":[{"localId":"2uGhB2WNfipw4eiiLiovjxzjY24f","createdAt":"1635148124461","lastLoginAt":"1635148124461","emailVerified":false,"email":"test@test.com","salt":"fakeSaltxvJVLUvcTpxVb1qzMmoP","passwordHash":"fakeHash:salt=fakeSaltxvJVLUvcTpxVb1qzMmoP:password=password","passwordUpdatedAt":1635148124461,"validSince":"1635148124","providerUserInfo":[{"providerId":"password","email":"test@test.com","federatedId":"test@test.com","rawId":"test@test.com","displayName":"test"}],"lastRefreshAt":"2021-10-25T07:48:44.462Z","displayName":"test"}]}
```
Comment (kylesome) [inline firebase-emulator/fixtures/sample-imageswipe-test/saved-data/auth_export/accounts.json:1]: might be good to pretty print this one

File: src/main.js [hunk 2]
```diff
@@ -20,9 +25,22 @@ This will make sure Firebase initializes before loading the app when a user refr
 - https://savvyapps.com/blog/definitive-guide-building-web-app-vuejs-firebase
 */
 
+let config = appConfig;
+// Sample setup for ImageSwipe and WordSwipe for development for testing only. Will be removed when testing is done
+if (process.env.VUE_APP_PROJECT_ID === "imageswipe-test") {
+  config = imageswipeTestConfig;
+}
+if (process.env.VUE_APP_PROJECT_ID === "wordswipe-test") {
```
Comment (kylesome) [inline src/main.js:33]: you could probably put this logic in the `config.js` and just have that file export the correct one

**Review by kylesome — APPROVED** [review]
Looks good, nice to have a repeatable setup for testing 👍

File: firebase-emulator/fixtures/sample-imageswipe-test/saved-data/auth_export/accounts.json (hunk 1 above)
Comment (sevresbabylone) [reply firebase-emulator/fixtures/sample-imageswipe-test/saved-data/auth_export/accounts.json:1]: Hey! Not sure what you mean by that. Would it be possible to give an example?

Comment (kylesome) [reply firebase-emulator/fixtures/sample-imageswipe-test/saved-data/auth_export/accounts.json:1]: hey, sorry for late reply, just meant not all on one line that's all, not critical, just a suggestion 🙂

**Pull request merged**